assignments/stretch-function-conversion.js

```diff
@@ -2,22 +2,38 @@
 
 // let myFunction = function () {};
 
+let myFunction = () => {};
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 
+let anotherFunction = (param) => {
+    return param;
+}
+
```

Comment:
Can be refactored to:
`let anotherFunction = param => param`
- parentheses are optional when you only have 1 parameter